sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java

```diff
@@ -283,6 +283,11 @@ private View prepareRootView(@NonNull LayoutInflater inflater,
                 return true;
             }
 
+            final boolean ignoreBackPress = Boolean.FALSE.equals(config.getStopVerificationOnBackPress());
+            if (ignoreBackPress) {
+                return true;
+            }
+
             return webViewHelper != null && webViewHelper.shouldRetry(
                     new HCaptchaException(HCaptchaError.CHALLENGE_CLOSED));
```

Comment:
Can't `config.getStopVerificationOnBackPress()` already be achieved with a custom `config.retryPredicate`?

Reply:
I think no.

`config.retryPredicate` is about retrying verification when it failed for some reason. `config.stopVerificationOnBackPress=false` about ignoring user's back presses/gestures

Reply:
Good point, PR updated accordingly